api/src/main/java/me/lokka30/treasury/api/economy/currency/Currency.java

```diff
@@ -157,9 +158,10 @@ default BigDecimal to(@NotNull Currency currency, @NotNull BigDecimal amount) {
         Objects.requireNonNull(currency, "currency");
         Objects.requireNonNull(amount, "amount");
 
-        return amount
-                .multiply(this.getConversionRate())
-                .divide(currency.getConversionRate(), RoundingMode.HALF_UP);
+        return amount.multiply(this.getConversionRate()).divide(
+                currency.getConversionRate(),
+                RoundingMode.HALF_UP
+        );
```

Comment:
this is just worse

Reply:
that's just how checkstyle did it.

Reply:
can't be bothered fighting checkstyle, will just let it have its way